src/components/DuosHeader.js

```diff
@@ -8,21 +8,59 @@ import MenuIcon from '@material-ui/icons/Menu';
 import {Link, withRouter} from 'react-router-dom';
 import {Storage} from '../libs/storage';
 import {SupportRequestModal} from './modals/SupportRequestModal';
-import './DuosHeader.css';
+import './DuosNavigation.css';
 import { NotificationService } from '../libs/notificationService';
 import { Notification } from './Notification';
 import { Styles } from '../libs/theme';
 import DuosLogo from '../images/duos-network-logo.svg';
 import contactUsHover from '../images/navbar_icon_contact_us_hover.svg';
 import contactUsStandard from '../images/navbar_icon_contact_us.svg';
-import { isNil, map, uniq } from 'lodash/fp';
+import { isNil } from 'lodash/fp';
 import {Config} from '../libs/config';
+import Tabs from '@mui/material/Tabs';
+import Tab from '@mui/material/Tab';
+import Box from '@mui/material/Box';
 
 const styles = {
   drawerPaper: {
     backgroundColor: '#00243c',
     color: 'white',
     fontFamily: 'Montserrat'
+  },
+  mainTab: {
+    padding: '0 25px',
+    fontSize: '16px',
+    textTransform: 'none',
+    fontFamily: '\'Montserrat\', sans-serif',
```

Comment:
nit: this also works:
```suggestion
    fontFamily: 'Montserrat, sans-serif',
```